### ISSUE TITLE
Corruption malus when equip ancient weapons or gods weapon

### ISSUE THREAD

**Issue by dessemon** [Tirocupidus/TheExiledRPOverhaul#92]
Hi :)

Here are some suggestions. 

First, the ancient weapons are supposed to corrupt their holders.
"A relic of the past, the shape of it calls up shadow-haunted memories of ape-gods squatting on shattered altars. It whispers ceaselessly to its bearer and slowly, inevitably, tears their sanity apart. "

**Suggestion :** Apply a corruption when character equip an ancient weapon.

Secondly, weapons dedicated to a god should apply a penalty to a character who is not a believer of that god.

**Suggestion :** Apply a corruption when character equip an god weapon when this god is not the god of character.

**Comment by gjpeebles** [comment]
The Ancient corruption has been recommended before, but [it had fallen](https://github.com/Tirocupidus/TheExiledRPOverhaul/issues/10) to the bottom of the list. :grin: I think it won't be a problem, but at the same time, I wanted to to make them not craftable, and increase their stats. And also replace them with a new item in the professions that depend on them, like a new T3 bow.

I also really like the idea to penalize a character who uses a religious weapon other than their own. ^^ I want to add all sorts of fun stuff for the priest professions, and while this could be a part of that, I should be able to add it in before I get into any bigger changes.

**Comment by dessemon** [comment]
Very good news. And I totally love the idea of not being able to craft ancient weapons.

**Comment by mlloy** [comment]
need a t3 bow replacement then otherwise why be an archer?

**Comment by dessemon** [comment]
On our side, the archers uses the Hykranian bow, and that seems to be ok. But may be is there others solutions :)

**Comment by mlloy** [comment]
In PvP, there is literally no means of being an archer when a black ice broadsword does 8x the damage.  It gates off both end-game PVP and PVE content for archers.  

Try killing a wolf with a hyrkanian bow in the North without spending 8-10 arrows.  That's alot of arrows for a wolf.

**Comment by CupcakeValkyrie** [comment]
I agree. Archers need a T4 option.

**Comment by gjpeebles** [comment]
There are bolts and arrows. In vanilla (and currently TERPO), the highest light damage an archer can get is Ancient Bow (38) + Star Metal Arrows (61) for 99 damage, which is actually more damage than a Star Metal Spear, albeit with only 32.5% armor penetration. You need to take into account ammunition before you say something does 8x more damage than a bow.

**Comment by mlloy** [comment]
No, I was comparing black ice broadsword with a Hyrkanian bow, which would be the only viable option for an archer once corruption is introduced to an ancient bow.  It's a huge nerf.  Now archers that want to be competitive will have ~50% less health and stamina than warriors, or be forced to use a Hyrkanian bow.  My suggestion was if you were going to implement a corruption malus, you should make the bow more powerful to adjust for the huge health and stamina nerf that comes with corruption.  Archers are already pretty bad vs. full strength warriors and this would just basically make one unplayable for both pve and pvp content (speaking as someone with a 0 str and 35 accuracy character).

Furthermore! Using star metal arrows as ammunition I don't think is a fair comparison, considering how quickly and easily one can burn through ammunition and how hard star metal is to get.

**Comment by gjpeebles** [comment]
>  I wanted to to make them not craftable, and increase their stats. And also replace them with a new item in the professions that depend on them, like a new T3 bow.

You seem to be saying what I've already said.

**Comment by mlloy** [comment]
Looks like it ya

**Comment by Shadowcrafting** [comment]
Had an idea! Instead of corrupting you.... you gain a damage bonus to half the % of corruption you have! Will you delve into long forgotten places to discover how to aligne yourself to your dreaded weapon, or resist the dark promise of power and maintain your humanity.... O_O

**Comment by CraftsmasterKubilai** [comment]
I actually like that idea about adding a damage bonus based on your current Corruption score - that way there's a true ability to balance "risk-vs-reward" in a player-controllable way. To keep it simple, I suggest starting with a setting that damage for the weapon is scaled up such that when the player is at maximum corruption (-50% to health and stamina), the weapon does 50% more damage, scaling up/down in 1% increments. Under this system, Ancient weapons should function exactly as they do now when the wielder has a zero corruption score.  I also agree that Ancient weapons should NOT be craftable in most circumstances (see below for a potential crafting alternative), and should, instead, be rare loot drops from boss-level mobs or boss chests.

Re: Replacement T4 weapons - I suggest we add a crafting tier that replaces "Ancient" weapons with "Peerless" versions. Peerless gear functions exactly like Ancient does now (same durability and damage), but does not have the new "damage scaling" properties described above regardless of the user's corruption score.

Re: Crafting Corrupted - We COULD allow smiths who are able to craft "Peerless" weapons to make "Ancient" (scaling) versions, but ONLY when the crafter in question has a maximum corruption score at the time the crafting event takes place. Under this system, when the smith has max corruption, the smith would use the "Peerless" weapon recipe to trigger the crafting event, but the resulting item produced is automatically replaced with the "Ancient" (scaling) version (they literally can't make non-Ancient weapons while their brains are seething with otherworldly knowledge). If/when the smith's corruption level drops to anything less-than full, only 'Peerless" versions will result.  We COULD also make this ability only available if the smith learns a special recipe, say by finding a book located in the Forbidden City (we'd need to add the item to the world) and reading it. I suggest that smiths reading the "Ancient Tome" described above be given an irreducible 10% corruption score (cannot be mitigated by dancers or yellow pots) to reflect the "price of dark knowledge" inflicted on the human mind for viewing such  thing... =)

**_Last, if it is decided to change weapons in this manner, please let me (Kubilai) know in advance - I actually have an in-game RP event idea that can introduce this change to the server and 'explain' it via story and lore if we want to go that route! I can also write the text of the "Ancient Tome" to match this RP/world event. _**

**Comment by gjpeebles** [comment]
I kinda prefer the idea that the weapon is already powerful, but using it comes with a cost. I'm curious where other people stand.

**Comment by Shadowcrafting** [comment]
I like the idea of you have to corrupt yourself to unlock the weapons full potential, otherwise you'll have people with pocket dancers that have high damage, but no corruption

**Comment by gjpeebles** [comment]
I will probably disable the dancer corruption heal while the weapon is being wielded, regardless of how else it works.
